From d85c94b89aa741d6d90cf0712e47d566cf107f8c Mon Sep 17 00:00:00 2001
From: RobertGarciaa <chae0218@naver.com>
Date: Fri, 26 Dec 2025 22:32:07 +0900
Subject: [PATCH] ro.lineage -> ro.pixelg

Signed-off-by: RobertGarciaa <chae0218@naver.com>
---
 .../LineageVendorSecurityPatchLevelPreference.kt                | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/com/android/settings/deviceinfo/firmwareversion/LineageVendorSecurityPatchLevelPreference.kt b/src/com/android/settings/deviceinfo/firmwareversion/LineageVendorSecurityPatchLevelPreference.kt
index 2781173d7da..5af91b32b7b 100644
--- a/src/com/android/settings/deviceinfo/firmwareversion/LineageVendorSecurityPatchLevelPreference.kt
+++ b/src/com/android/settings/deviceinfo/firmwareversion/LineageVendorSecurityPatchLevelPreference.kt
@@ -71,6 +71,6 @@ class LineageVendorSecurityPatchLevelPreference :
         const val AOSP_VENDOR_SECURITY_PATCH_PROPERTY: String =
             "ro.vendor.build.security_patch"
         const val LINEAGE_VENDOR_SECURITY_PATCH_PROPERTY: String =
-            "ro.lineage.build.vendor_security_patch"
+            "ro.pixelg.build.vendor_security_patch"
     }
 }
-- 
2.43.0

