From 2bd6c1f7ea7606df8210df8561da7f4c9017a7de Mon Sep 17 00:00:00 2001
From: RobertGarciaa <chae0218@naver.com>
Date: Fri, 10 Jul 2026 13:13:33 +0900
Subject: [PATCH] ro.pixelg

Signed-off-by: RobertGarciaa <chae0218@naver.com>
---
 common/private/property_contexts | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/common/private/property_contexts b/common/private/property_contexts
index fb5e162..885d510 100644
--- a/common/private/property_contexts
+++ b/common/private/property_contexts
@@ -25,6 +25,13 @@ ro.crdroid.build.version               u:object_r:custom_version_prop:s0
 ro.crdroid.display.version             u:object_r:custom_version_prop:s0
 ro.crdroid.version                     u:object_r:custom_version_prop:s0
 
+# Custom version (pixelg)
+ro.pixelg.device                       u:object_r:custom_version_prop:s0
+ro.pixelg.version                      u:object_r:custom_version_prop:s0
+ro.pixelg.display.version              u:object_r:custom_version_prop:s0
+ro.pixelg.build.version                u:object_r:custom_version_prop:s0
+ro.pixelg.releasetype                  u:object_r:custom_version_prop:s0
+
 # Custom version (lineage)
 ro.lineage.version                     u:object_r:custom_version_prop:s0
 ro.lineage.display.version             u:object_r:custom_version_prop:s0
-- 
2.54.0

