From 24a3f1c01c7e6e39a7356d099c12d1c9837a812a Mon Sep 17 00:00:00 2001
From: RobertGarciaa <chae0218@naver.com>
Date: Thu, 16 Apr 2026 11:49:41 +0900
Subject: [PATCH] fix mindthegapps issue

---
 config/common.mk | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/config/common.mk b/config/common.mk
index 3ef7c500..b933a551 100644
--- a/config/common.mk
+++ b/config/common.mk
@@ -266,11 +266,6 @@ endif
 # Audio files
 $(call inherit-product, vendor/lineage/audio/audio.mk)
 
-# SetupWizard
-PRODUCT_PRODUCT_PROPERTIES += \
-    setupwizard.theme=glif_expressive \
-    setupwizard.feature.day_night_mode_enabled=true
-
 PRODUCT_ENFORCE_RRO_EXCLUDED_OVERLAYS += vendor/lineage/overlay/no-rro
 PRODUCT_PACKAGE_OVERLAYS += \
     vendor/lineage/overlay/common \
-- 
2.43.0

