From 1dd21fe3aba85db8c0deef6a32297c04e3d33ca3 Mon Sep 17 00:00:00 2001
From: RobertGarciaa <chae0218@naver.com>
Date: Fri, 26 Dec 2025 22:36:13 +0900
Subject: [PATCH] fix mindthegapps issue

Signed-off-by: RobertGarciaa <chae0218@naver.com>
---
 config/common.mk | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/config/common.mk b/config/common.mk
index b45b10dc..5c36b147 100644
--- a/config/common.mk
+++ b/config/common.mk
@@ -260,11 +260,6 @@ endif
 # Audio files
 $(call inherit-product, vendor/lineage/audio/audio.mk)
 
-# SetupWizard
-PRODUCT_PRODUCT_PROPERTIES += \
-    setupwizard.theme=glif_expressive \
-    setupwizard.feature.day_night_mode_enabled=true
-
 PRODUCT_ENFORCE_RRO_EXCLUDED_OVERLAYS += vendor/lineage/overlay/no-rro
 PRODUCT_PACKAGE_OVERLAYS += \
     vendor/lineage/overlay/common \
-- 
2.43.0

